applications/post-editing.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Human editing of machine translation output
+---
+
+# Post-editing
+
+Human **Post-editing** is the review and correction of machine translation output. When post-editing, the human translator checks the original text and fixes any error made by the machine translation.
+
+Post-editing is a replacement of human translation as it has faster turnaround times, is cheaper, and can process larger amounts of information than the traditional human translation process.
```

Comment:
Post-editing started to replace human translation because it is faster and cheaper than the traditional human translation process.